docker/perception/lane_detection/lane_detection.Dockerfile

```diff
@@ -43,13 +54,28 @@ RUN . /opt/ros/$ROS_DISTRO/setup.sh && \
 
 # Entrypoint will run before any CMD on launch. Sources ~/opt/<ROS_DISTRO>/setup.bash and ~/ament_ws/install/setup.bash
 COPY docker/wato_ros_entrypoint.sh ${AMENT_WS}/wato_ros_entrypoint.sh
-ENTRYPOINT ["./wato_ros_entrypoint.sh"]
+# ENTRYPOINT ["./wato_ros_entrypoint.sh]
 
 ################################ Prod ################################
-FROM build as deploy
+FROM ${RUNTIME_IMAGE} as deploy
```

Comment:
If a person runs this image as a dev container, can he/she rebuild the lane detection node inside it? eg. run colcon build while inside the container

Reply:
oh, nvm I already addressed this 😅